### ISSUE TITLE
callbacks ?

### ISSUE THREAD

**Issue by matrixreal** [appodeal/appodeal-cordova-plugin#26]
HI,
i would like to know how to disable completely the callbacks ?
i got much cpu usage and i would like to remove them completely
thank you

**Comment by matrixreal** [comment]
any news ?

**Comment by meeeeeepo** [comment]
@matrixreal hello, for iOS callbacks are disabled by default, so if you haven't enabled them manually then they are not active

**Comment by matrixreal** [comment]
but i see them in the log with simulator and with real device

**Comment by meeeeeepo** [comment]
what exactly do you see in log? could you please paste it here?

**Comment by matrixreal** [comment]
![appodeal](https://cloud.githubusercontent.com/assets/6823974/24282951/fc958014-1062-11e7-9a42-129d2d911738.jpg)

**Comment by meeeeeepo** [comment]
that is not callback log, it's just appodeal SDK logs

**Comment by matrixreal** [comment]
ok yes how to disable them please
thank you

**Comment by meeeeeepo** [comment]
you can call Appodeal.setLogging(false) before initialising appodeal

**Comment by matrixreal** [comment]
ok thank you
will you update admod sdk soon?

**Comment by meeeeeepo** [comment]
yeah, we will soon update all the networks inside our SDK, can't say the exact date for now

**Comment by matrixreal** [comment]
ok
last question what do this call exactly? Appodeal.setLogging(false);

**Comment by matrixreal** [comment]
sorry this one  Appodeal.disableLocationPermissionCheck();

**Comment by meeeeeepo** [comment]
this method disables user location tracking for appodeal SDK, this method will be updated with next release of plugin

**Comment by matrixreal** [comment]
@meeeeeepo 
ok thank you
what are callbacks exactly ? what is the use of  ?

**Comment by meeeeeepo** [comment]
@matrixreal callbacks are needed to catch some events occuring in appodeal SDK, like loading ads or showing them
You can read about it here: https://www.appodeal.com/sdk/documentation?framework=9&full=1&platform=4#p_4_6

**Comment by matrixreal** [comment]
@meeeeeepo so it's better to disable them once we build the ipa for distribution ?

**Comment by meeeeeepo** [comment]
@matrixreal It's all up to you, usually developers use callbacks to do something when needed event occured, for example if you are showing rewarded or skippable video you can give a reward to player when video was finished,

**Comment by NazarKozak** [comment]
@matrixreal is it working for you now?
